projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -32,10 +31,10 @@ export class PageTimeRangePreferenceService {
   public getTimeRangePreferenceForPage(rootLevelPath: string): Observable<TimeRangeResolver> {
     return combineLatest([
       this.pageTimeRangeStringDictionary$,
-      this.featureStateResolver.getFeatureState(ApplicationFeature.PageTimeRange)
+      this.featureStateResolver.getFeatureFlagValue<boolean>(ApplicationFeature.PageTimeRange)
```

Comment:
this should still be calling `getFeatureState`, it doesn't need the raw value

Reply:
fixed